include/oneapi/dpl/internal/dynamic_selection.h

```diff
@@ -0,0 +1,205 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include <memory>
+#include <utility>
+#include <iostream>
+#include <sstream>
+
+#ifndef _ONEDPL_DYNAMIC_SELECTION_DEFS_H
+#define _ONEDPL_DYNAMIC_SELECTION_DEFS_H
+#pragma once
+
+
+namespace oneapi {
+namespace dpl {
+namespace experimental {
+//ds_properties
+
+  namespace property {
+    struct universe_t {
+      static constexpr bool is_property_v = true;
+      static constexpr bool can_report_v = false;
+    };
+    inline constexpr universe_t universe;
```

Comment:
If universe is the only thing we need for the 1st release, can we drop generic properties altogether and just require a method for this on the scheduler type?  Maybe we can just add setters and getters.

Reply:
Aside the universe and universe size properties, all others have been dropped